cmd/clusterctl/client/cluster/objectgraph.go

```diff
@@ -194,15 +199,21 @@ func (o *objectGraph) objToNode(obj *unstructured.Unstructured) *node {
 		tenantClusters: make(map[*node]empty),
 		tenantCRSs:     make(map[*node]empty),
 		virtual:        false,
-		forceMove:      o.getForceMove(obj.GetKind(), obj.GetAPIVersion()),
+		forceMove:      o.getForceMove(obj.GetName(), obj.GetNamespace(), obj.GetKind(), obj.GetAPIVersion(), obj.GetLabels()),
 		isGlobal:       isGlobal,
 	}
 
 	o.uidToNode[newNode.identity.UID] = newNode
 	return newNode
 }
 
-func (o *objectGraph) getForceMove(kind string, apiVersion string) bool {
+func (o *objectGraph) getForceMove(name, ns, kind, apiVersion string, labels map[string]string) bool {
+	if _, ok := labels[clusterctlv1.ClusterctlMoveLabelName]; ok {
+		log := logf.Log
+		log.V(5).Info("Force move object", "kind", kind, "name", name, "namespace", ns)
```

Comment:
I'm for removing this log, given that with the dryrun feature there are other means to detect what is included in move scope